openwhisk/openwhisk_apigateway.md

```diff
@@ -80,48 +133,61 @@ You have a series of actions to implement your backend for the book club:
 
 Let's create an API for the book club, named `Book Club`, with `/club` as its HTTP URL base path and `books` as its resource.
 ```
-wsk api-experimental create -n "Book Club" /club /books get getBooks
-wsk api-experimental create /club /books post postBooks
-wsk api-experimental create /club /books put putBooks
-wsk api-experimental create /club /books delete deleteBooks
+wsk api create -n "Book Club" /club /books get getBooks --response-type http
+wsk api create /club /books post postBooks              --response-type http
+wsk api create /club /books put putBooks                --response-type http
+wsk api create /club /books delete deleteBooks          --response-type http
 ```
-{: pre}
 
 Notice that the first action exposed with base path `/club` gets the API label with name `Book Club` any other actions exposed under `/club` will be associated with `Book Club`
 
 Let's list all the actions that we just exposed.
 
 ```
-wsk api-experimental list
+wsk api list -f
+```
+```
+ok: APIs
+Action: getBooks
+  API Name: Book Club
+  Base path: /club
+  Path: /books
+  Verb: get
+  URL: https://service.us.apiconnect.ibmcloud.com/gws/apigateway/api/21ef035/club/books
+Action: postBooks
+  API Name: Book Club
+  Base path: /club
+  Path: /books
+  Verb: post
+  URL: https://service.us.apiconnect.ibmcloud.com/gws/apigateway/api/21ef035/club/books
+Action: putBooks
+  API Name: Book Club
+  Base path: /club
+  Path: /books
+  Verb: put
+  URL: https://service.us.apiconnect.ibmcloud.com/gws/apigateway/api/21ef035/club/books
+Action: deleteBooks
+  API Name: Book Club
+  Base path: /club
+  Path: /books
+  Verb: delete
+  URL: https://service.us.apiconnect.ibmcloud.com/gws/apigateway/api/21ef035/club/books
 ```
-{: pre}
-```
-ok: apis
-Action                   Verb          API Name        URL
-getBooks                   get         Book Club       https://2ef15285-gws.api-gw.mybluemix.net/club/books
-postBooks                 post         Book Club       https://2ef15285-gws.api-gw.mybluemix.net/club/books
-putBooks                   put         Book Club       https://2ef15285-gws.api-gw.mybluemix.net/club/books
-deleteBooks             delete         Book Club       https://2ef15285-gws.api-gw.mybluemix.net/club/books
-```
-{: screen}
 
 Now just for fun let's add a new book `JavaScript: The Good Parts` with a HTTP __POST__
 ```
-curl -X POST -d '{"name":"JavaScript: The Good Parts", "isbn":"978-0596517748"}' https://2ef15285-gws.api-gw.mybluemix.net/club/books
+curl -X POST -d '{"name":"JavaScript: The Good Parts", "isbn":"978-0596517748"}' https://service.us.apiconnect.ibmcloud.com/gws/apigateway/api/21ef035/club/books
 ```
-{: pre}
 ```
 {
   "result": "success"
 }
 ```
```

Comment:
oh.. does the postBooks action explicitly return `{"result":"success"}` in the http reponse `body` ?